### ISSUE TITLE
Repeat apply on fail_over_mac: "1" bond failed with NM_ACTIVE_CONNECTION_STATE_REASON_DEVICE_DISCONNECTED

### ISSUE THREAD

**Issue by cathay4t** [nmstate/nmstate#781]
2020-02-12 20:38:24,131 root         ERROR    NM main-loop aborted: Connection activation failed on bond99: reason=<enum NM_ACTIVE_CONNECTION_STATE_REASON_DEVICE_DISCONNECTED of type NM.ActiveConnectionStateReason>, <enum NM_DEVICE_STATE_FAILED of type NM.DeviceState> <enum NM_DEVICE_STATE_REASON_NONE of type NM.DeviceStateReason>

When applying this ymls repeatly(normally, second `set` will fail):

```yml
---
interfaces:
- name: eth1
  type: ethernet
  state: up
- name: bond99
  type: bond
  state: up
  link-aggregation:
    mode: active-backup
    slaves:
    - eth1
    options:
      fail_over_mac: "1"
```

https://bugzilla.redhat.com/show_bug.cgi?id=1801798

**Comment by cathay4t** [comment]
It's a NetworkManager's bug:
[Bug 1802424][1] - Got failure when activating bond with fail_over_mac=1 option. 

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1802424

**Comment by tyll** [comment]
It seems that Nmstate should just not try to persist the MAC address if `fail_over_mac` is set, see https://bugzilla.redhat.com/show_bug.cgi?id=1802424#c3

**Comment by tyll** [comment]
Also Nmstate should raise a ValueError if the desired state specifies a MAC address that that does not match the current state (or maybe actually the first slave's MAC address).

**Comment by ffmancera** [comment]
Could we close this? #827 @cathay4t

**Comment by cathay4t** [comment]
Closing as fixed been included in master and v0.2.6
